src/NzbDrone.Core/Datastore/Migration/074_disable_eztv.cs

```diff
@@ -0,0 +1,14 @@
+﻿using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(74)]
+    public class disable_eztv : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Execute.Sql("UPDATE Indexers SET EnableRss = 0, EnableSearch = 0 WHERE Implementation = 'Eztv' AND Settings LIKE '%ezrss.it/%'");
```

Comment:
I went with handwritten inline SQL because the Update fluent syntax didn't appear to support the LIKE statement, plus we has tests.